### ISSUE TITLE
Newspaper: Can not install with Python 2.7.12

### ISSUE THREAD

**Issue by masonshuler** [mmw125/MuDimA#59]
https://github.com/codelucas/newspaper/issues/471

Don't know of any solutions. My AWS deployment is on halt until this solution is fixed. Is anyone else getting this issue?

**Comment by jteves** [comment]
I've had some issues when I installed it on a virtualbox with Ubuntu. I
just tried different Stack Overflow stuff like:
https://stackoverflow.com/questions/6504810/how-to-install-lxml-on-ubuntu?noredirect=1&lq=1

**Comment by 10B-B11** [comment]
@masonshuler do you find any solutions?

**Comment by masonshuler** [comment]
Not yet. I am planning on installing newspaper from source with a custom
requirements.txt

**Comment by 10B-B11** [comment]
@masonshuler how to do it？ i' m new pythoner

**Comment by masonshuler** [comment]
@caiyajing9210 
What I did was:
I cloned the newspaper git and branched to the python-2-head.
Next is I changed all of the == on the requirements.txt to >=
Then I pip installed the requirements. Along with my other requirements.
I moved all of the newspaper folder (the one containing __init__.py) to the folder with my newspaper dependency. 
Everything worked fine from my knowledge after then.

Hope this helps!
